make/RunTestsPrebuiltSpec.gmk

```diff
@@ -176,3 +176,7 @@ ULIMIT := ulimit
 ifeq ($(OPENJDK_BUILD_OS), windows)
   PATHTOOL := cygpath
 endif
+
+ifneq ($(TIDY_HOME), )
+  TIDY := $(TIDY_HOME)/bin/tidy
+endif
```

Comment:
Unless TIDY_HOME is a well established variable in the context of Tidy, which I doubt, then I think this isn't needed anymore. We aren't setting this from jib-profiles.js, and I can't think of any other place this would be set. We should export `TIDY` as an environment variable in the `run-test-prebuilt-docs` profile in the other PR though.

Reply:
I need to push one more patch in the other PR for these to be in sync but `TIDY_HOME` is needed.

`RunTestsPrebuiltSpec.gmk` is only read if you use `make run-test-prebuilt`, without running configure. (I'm stealing Magnus' words here)